caikit_nlp/config/config.yml

```diff
@@ -23,7 +23,3 @@ source_prompt_base: ""
 
 runtime:
   library: caikit_nlp
-  service_generation:
```

Comment:
This config was removed in an earlier caikit version: https://github.com/caikit/caikit/releases/tag/v0.7.0